### DISCUSSION TITLE
Is there any way to subscribe to the "collections" table?

### DISCUSSION THREAD

**Discussion by ahnaf-asif** [pocketbase/pocketbase#3706] [Q&A]
Hello everyone! As pocketbase gives us SSE functionalities where we can just subscribe to a collection and then we get notifications from server whenever the collection is updated:

```js
pb.collection('a_collection').subscribe('*', function (e) {
    // do something
  });
```

Now, there is also a way to get the list of all collections from pocketbase, using something like this: `await pb.collections.getFullList();`. Now, is it possible to subscribe to pb.collections where whenever a new collection is created or any existing ones updated, I can get real time notifications for them? Thanks in advance

**Comment by ganigeorgiev** [comment]
No, there is no builtin functionality for that at the moment.

**Reply by ahnaf-asif** [reply]
Is there any way to accomplish that? Please let me know. Thanks in advance!

**Reply by ganigeorgiev** [reply]
It can be implemented with a custom subscription topic but without details about your use case and why you need it, I don't feel confident recommending it.

**Reply by ganigeorgiev** [reply]
@gedw99 For now there are no plans to introduce web realtime events for the collection schema changes.

If you want to listen to collection model changes you can attach with the Go (or JSVM) db hooks - https://pocketbase.io/docs/go-event-hooks/#db-hooks, for example:

```go
app.OnModelAfterCreate("_collections").Add(func(e *core.ModelEvent) error {
	collection := e.Model.(*models.Collection)
	...

	return nil
})
```
